ipn/ipnlocal/web_client.go

```diff
@@ -88,3 +94,38 @@ func (b *LocalBackend) handleWebClientConn(c net.Conn) error {
 	s := http.Server{Handler: b.webClient.server}
 	return s.Serve(netutil.NewOneConnListener(c, nil))
 }
+
+func (b *LocalBackend) updateWebClientTCPPortNetMapAddrListenersLocked() {
+	if b.netMap == nil {
+		return
+	}
+
+	addrs := b.netMap.GetAddresses()
+	for i := range addrs.LenIter() {
+		addrPort := netip.AddrPortFrom(addrs.At(i).Addr(), webClientPort)
+		if _, ok := b.webClientListeners[addrPort]; ok {
+			continue // already listening
+		}
+
+		sl := b.newWebClientListener(context.Background(), addrPort, b.logf)
+		mak.Set(&b.webClientListeners, addrPort, sl)
+
+		go sl.Run()
+	}
+}
+
+// newWebClientListener returns a listener for local connections to the built-in web client
+// used to manage this tailscale instance.
```

Comment:
Tailscale

give an example of the address it's listening on? This is for http://Tailscale-IP:1234/ or 100.100.100.100 or?

Reply:
fixed capitalization.  Added more detail on which IP/port combinations are listened on the docs for `updateWebClientListenersLocked`